source/features/linkify-branch-refs.tsx

```diff
@@ -6,34 +6,34 @@ import {getOwnerAndRepo} from '../libs/utils';
 import {safeElementReady, wrap} from '../libs/dom-utils';
 
 function inPR() {
-	let deletedBranch: string;
+	let deletedBranch: string | undefined;
```

Comment:
Why not `string?`

Reply:
TypeScript complains about accessing a variable before its set. This is fine when not in strict mode, but as you tighten the screws...